docs/docs/reference/package-log.mdx

```diff
@@ -75,7 +75,7 @@ logger.Error("Message")
 #### Usage
 
 ```go
-if err := logger.Option(log.WithContextFieldsResolver(log.ContextLoggerFieldsResolver)); err != nil {
+if err := logger.Option(log.WithContextFieldsResolver(log.MergedContextLoggerFieldsResolver)); err != nil {
```

Comment:
`log.MergedContextLoggerFieldsResolver`  
Does this have to be renamed?  
At least I can't find it in the code.